recipes/fsspec-reference-maker/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "fsspec-reference-maker" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b40f88718f93508135a8b9e839c0d7880317080a6a26f1128d610e0bc8952f3b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.7
+    - fsspec
+    - ujson
+    - numpy
+    - xarray
+    - zarr
+    - packaging
```

Comment:
Just FYI, while I see these are imported by the code, they aren't listed in your upstream `requirements.txt`. That means your PyPI package metadata isn't listing them as dependencies, so `pip install` for this package is probably broken (unless you have all those packages already).

Reply:
I am following the conda-forge "kitchen sink" model: there are the packages that are very likely to be needed, but you can import the top level package without them. In the pypi package, they should be included in the `extras_require`, and probably will be soon.